serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
 
-* Define a channel for certain Cloud Event Types and let the not mapped Cloud Event types to use the default incoming or outgoing channel.
+The default incoming channel is `kogito_incoming_stream` and the default outgoing channel is `kogito_outgoing_stream`.
 
-Kogito first searches for channel name equals to Cloud Event type in the properties. If found, it uses that channel for that Cloud Event type. If not, it searches for default channel definition. If also not existing, then an error will be reported.
-
-The name for the default incoming channel is `kogito_incoming_stream` and for the default outgoing channel is `kogito_outgoing_stream`.
-
-You can change the default incoming and outgoing topic name as follows:
+To change the default incoming and outgoing topic names, you can use the following properties:
 
+.Properties to change default incoming and outgoing topic names
 [source,properties]
 ----
 kogito.addon.messaging.incoming.defaultName=<default channel name>
 kogito.addon.messaging.outgoing.defaultName=<default channel name>
 ----
 
-This is the full list of link:{kafka_connector_properties_url}[properties] you can use for a channel when using Kafka connector.
+The following properties are mandatory that you need to set for each channel:
 
-There are two mandatory properties that you need to setup for every channel defined:
+* `connector`: This property needs to be set to `smallrye-kafka`.
+* Depending on whether the channel is incoming or outgoing:
+** `value.deserializer`: This property is used for incoming channels. Unless you have specific marshaling requirements, you must set this property to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`.
+** `value.serializer`: This property is used for outgoing channels. Unless you have specific marshaling requirements, you must set this property to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`.
 
-* `connector`, which has to be set to `smallrye-kafka`
-* Depending on the channel being input or output: 
-** if incoming, `value.deserializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`
-** if outgoing, `value.serializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`
+[TIP]
+====
+`topic` is another relevant but optional property that needs to be set for a channel. The `topic` property must contain the Apache Kafka topic name to be used for a channel. If `topic` property is not set, then the channel name is used as topic name. 
```

Comment:
since this property is optional, rather than "needs to be set", it should read "might be usually set"

Reply:
So the sentence, in my opinion should be
`topic` is another relevant but optional property that might be set for a channel. The `topic` property contains the Apache Kafka topic name to be used for a channel. If `topic` property is not set, then the channel name is used as topic name.

Reply:
Done